src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -212,6 +212,7 @@
      * The current version of {@link Funcotator}.
      */
     public static final String VERSION = "0.0.2";
+    public static final int FEATURE_QUERY_LOOKAHEAD = 100000;
```

Comment:
Do you want to reference one of the other predefined constants here?

Reply:
Also, do you want a different value for Gencode vs. other feature inputs? Should we expose a command-line arg at the `GATKTool` and/or `VariantWalker` level?

Reply:
I think we might want different values for gencode... I'm not sure though.  I think maybe we can leave that decision to the future?

Reply:
I think a good option would be to put the look ahead value as a datasource property, that way each one could customize it.

Reply:
@jonn-smith Any thoughts on that?

Reply:
For now I've set it to use the same value of 100_000, but I'm referring to the value in `VariantWalkerBase`.  It looks like all variant walker features, both driving and side inputs are set to 100k.

Reply:
@jonn-smith I'm not sure I understand what you mean by making it a parameter on the datasource type.  Would we have a different parameter for each datasource type?  I'm going to merge this now and we can revisit if we need more granular control I think.